### PR TITLE
Identity manager/authorize validation

### PR DESCRIPTION
**Pull request by TadevosStepanyan**

### CODE REVIEW

File: src/Shared/IdentityClient/Config/ConfigServicesIdentityService.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        //     Registers services required by authentication services with OpenID Connect.
+        //
+        // Parameters:
+        //   services:
+        //     The Microsoft.Extensions.DependencyInjection.IServiceCollection.
+        //
+        // Returns:
+        //     A Microsoft.Extensions.DependencyInjection.IServiceCollection that can be used
+        //     to further configure Service.
+        public static IServiceCollection ConfigureAuthenticationForOIDC(this IServiceCollection services)
```
Comment (Soarc) [inline src/Shared/IdentityClient/Config/ConfigServicesIdentityService.cs:53]: We don't need this

File: src/Shared/IdentityClient/Travely.Shared.IdentityClient.Authorization.csproj [hunk 2]
```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+	<FrameworkReference Include="Microsoft.AspNetCore.App" />
+
+	<PackageReference Include="IdentityServer4" />
```
Comment (Soarc) [inline src/Shared/IdentityClient/Travely.Shared.IdentityClient.Authorization.csproj:10]: Don't need this reference

File: src/Shared/Packages.props [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (Soarc) [inline src/Shared/Packages.props:1]: We don't need this for shared projects

File: src/Shared/version.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
```
Comment (Soarc) [inline src/Shared/version.json:2]: Same

File: IdentityManager.sln [hunk 5]
```diff
@@ -1,45 +1,54 @@
-﻿
-Microsoft Visual Studio Solution File, Format Version 12.00
```
Comment (Soarc) [inline IdentityManager.sln:2]: Why IdentityManager files are are changed? Please revert them

**Review by Soarc — CHANGES_REQUESTED** [review]
I didn't see HttpRequest extension methods and Service-Service authentication and user identity forwarding files.

It is fine to add service-service authentication and Identity forwarding in separate PR

File: IdentityManager.sln (hunk 5 above)
Comment (TadevosStepanyan) [reply IdentityManager.sln:2]: I do not know why git is saying that there is change here I even copped it from identity-manager/dev and pushed it

File: src/Shared/IdentityClient/Config/ConfigIdentityService.cs [hunk 6]
```diff
@@ -0,0 +1,37 @@
+﻿
+using Microsoft.AspNetCore.Builder;
+
+namespace Travely.IdentityClient.Authorization
+{
+    public static class ConfigIdentityService
```
Comment (Soarc) [inline src/Shared/IdentityClient/Config/ConfigIdentityService.cs:6]: rename to IdentityApplicationBuilderExtensions

File: src/Shared/IdentityClient/Config/ConfigIdentityService.cs [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+        //     must appear between the calls to app.UseRouting() and app.UseEndpoints(...) for
+        //     the middleware to function correctly.
+        //
+        // Parameters:
+        //   app:
+        //     The Microsoft.AspNetCore.Builder.IApplicationBuilder to add the middleware to.
+        //
+        // Returns:
+        //     A reference to the IApplicationBuilder instance after the operation has completed.
+        public static IApplicationBuilder ConfigureAuthentication(this IApplicationBuilder app)
```
Comment (Soarc) [inline src/Shared/IdentityClient/Config/ConfigIdentityService.cs:28]: Rename to `ConfigureTravelyAuthentication` to no conflict with standart one.

**Review by Soarc — APPROVED** [review]

**Pull request merged**